data/human/ships.txt

```diff
@@ -49,15 +49,17 @@ ship "Aerie"
 		"X3200 Ion Steering"
 		"Hyperdrive"
 		
-	engine -10 91
-	engine 10 91
-	gun -13 -79 "Sidewinder Missile Launcher"
-	gun 13 -79 "Sidewinder Missile Launcher"
-	turret 0 0 "Heavy Anti-Missile Turret"
-	turret -17 14 "Heavy Laser Turret"
-	turret 17 14 "Heavy Laser Turret"
-	bay "Fighter" -42 -2
-	bay "Fighter" 42 2
+	engine 15 97
+	engine -15 97
+	engine 34.5 64
+	engine -34.5 64
```

Comment:
I hope this isn't backwards
```suggestion
	engine 15 97 .9
	engine -15 97 .9
	engine 34.5 64 .4
	engine -34.5 64 .4
```

Reply:
So I totally uploaded 2 engine only renders right before you commented this, whoops. If you want to keep it with the 2 that's totally fine, if you want me to re-do the 4 engines let me know, sorry for the confusion!